code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -223,6 +223,14 @@
 	spent_icon = "trackingshell-spent"
 	projectile_type = /obj/item/projectile/bullet/tracking
 
+/obj/item/ammo_casing/shotgun/moghes
+	name = "wall shell"
+	desc = "The appropriately-named wall shell, made specifically for the Moghesian wall gun."
+	desc_extended = "This complicated and expensive shotfun shell fires both a solid slug and a hail of shots at the same time, making it a destructive and dangerous ammunition at all ranges, assuming the user can point in the target's general direction. Somehow it seems to somewhat fit in other shotgun type weaponry too."
```

Comment:
Misspell, shot**fun** instead of shot**gun**

Reply:
i mean, it is fun too probably.